<!DOCTYPE html>
<html lang="en">
<head>
  
  
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    
    
    <link rel="shortcut icon" href="../img/favicon.ico">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
    <title>Analysis of Post-Translational Modifications - MassIVE Documentation</title>
    <link href="../css/bootstrap-3.3.7.min.css" rel="stylesheet">
    <link href="../css/font-awesome-4.7.0.css" rel="stylesheet">
    <link href="../css/base.css" rel="stylesheet">
    <link rel="stylesheet" href="../css/highlight.css">
    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
    <![endif]-->

    <script src="../js/jquery-3.2.1.min.js"></script>
    <script src="../js/bootstrap-3.3.7.min.js"></script>
    <script src="../js/highlight.pack.js"></script>
    
    <base target="_top">
    <script>
      var base_url = '..';
      var is_top_frame = false;
        
        var pageToc = [
          {title: "Analysis of Post-Translational Modifications using CCMS tools", url: "#_top", children: [
              {title: "Analyzing known modifications", url: "#analyzing-known-modifications" },
              {title: "Analyzing peptidoforms", url: "#analyzing-peptidoforms" },
              {title: "Analyzing hypermodified regions", url: "#analyzing-hypermodified-regions" },
              {title: "Analyzing putative novel modifications", url: "#analyzing-putative-novel-modifications" },
          ]},
        ];

    </script>
    <script src="../js/base.js"></script>
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-8412213-5', 'ccms-ucsd.github.io');
        ga('send', 'pageview');
    </script> 
</head>

<body>
<script>
if (is_top_frame) { $('body').addClass('wm-top-page'); }
</script>



<div class="container-fluid wm-page-content">
  <a name="_top"></a>
    

    
    
      
    

  <div class="row wm-article-nav-buttons" role="navigation" aria-label="navigation">
    
    <div class="wm-article-nav pull-right">
      <a href="../workflows/maestro/" class="btn btn-xs btn-default pull-right">
        Next
        <i class="fa fa-chevron-right" aria-hidden="true"></i>
      </a>
      <a href="../workflows/maestro/" class="btn btn-xs btn-link">
        Maestro workflow
      </a>
    </div>
    
    <div class="wm-article-nav">
      <a href="../search_identifications/" class="btn btn-xs btn-default pull-left">
        <i class="fa fa-chevron-left" aria-hidden="true"></i>
        Previous</a><a href="../search_identifications/" class="btn btn-xs btn-link">
        Search Identifications
      </a>
    </div>
    
  </div>

    

    <h1 id="analysis-of-post-translational-modifications-using-ccms-tools">Analysis of Post-Translational Modifications using CCMS tools</h1>
<p>We use two different workflows for identification of modifications</p>
<ul>
<li>
<p><a href="../workflows/maestro/">Maestro workflow</a>: identification of peptide-spectrum matches (PSMs) by i) spectral library search (MSPLIT) for detection of known reference peptides, followed by ii) restricted database search with known modifications (MSGF+), and finally followed by iii) open-modification search (MODa) for identification of peptides with unexpected modifications.</p>
<ul>
<li>Maestro primarily identifies modified <strong>peptide variants</strong>: distinct pairs (S,M) where S is a peptide sequence and M is the summed mass of all 'delta-mass' modifications assigned on sequence S. This abstraction is required for proper control of false discovery rates at PSM and peptide levels in Maestro; the resolution of delta-masses into putative modifications and modification sites is done with ModDecode as described below.</li>
</ul>
</li>
<li>
<p><a href="../workflows/moddecode/">ModDecode workflow</a>: post-processing of modified PSMs for i) resolution of delta-masses into known UniMod modifications for PSMs with unexpected modifications, ii) site assignment at fixed false localization rate and iii) calculation of AlignGF p-values between differently-modified versions of the same peptide.</p>
<ul>
<li>ModDecode primarily identifies modified <strong>peptidoforms</strong>: peptide sequences with site-resolved modifications whose masses were matched to known UniMod entries for putative resolution into (possibly combinations of) known modifications or amino acid polymorphisms.</li>
</ul>
</li>
</ul>
<p>The following sections focus on different aspects of the analysis of modifications and modified peptides. All links below use two example jobs whose results will be reported soon: a <a href="https://proteomics2.ucsd.edu/ProteoSAFe/status.jsp?task=de809a3717cc4d96a3a5257e290351d5">Maestro job</a> co-analyzing colon samples from the draft proteomes and a colon cancer dataset and the corresponding <a href="https://proteomics2.ucsd.edu/ProteoSAFe/status.jsp?task=790e82c2ef1541748521db1e94b24fe0">ModDecode</a> job with resolved modifications and peptidoforms.</p>
<h3 id="analyzing-known-modifications">Analyzing known modifications</h3>
<ol>
<li>Start with ModDecode list of <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_knownmod">UniMod modifications</a>; the default sorting order is by decreasing number of distinct peptides identified with each modification. </li>
<li>Additional important criteria for inspection of modifications:
    1.1. test sub-item in numbered lists 1</li>
</ol>
<h3 id="analyzing-peptidoforms">Analyzing peptidoforms</h3>
<h3 id="analyzing-hypermodified-regions">Analyzing hypermodified regions</h3>
<h3 id="analyzing-putative-novel-modifications">Analyzing putative novel modifications</h3>

  <br>
    

    
    
      
    

  <div class="row wm-article-nav-buttons" role="navigation" aria-label="navigation">
    
    <div class="wm-article-nav pull-right">
      <a href="../workflows/maestro/" class="btn btn-xs btn-default pull-right">
        Next
        <i class="fa fa-chevron-right" aria-hidden="true"></i>
      </a>
      <a href="../workflows/maestro/" class="btn btn-xs btn-link">
        Maestro workflow
      </a>
    </div>
    
    <div class="wm-article-nav">
      <a href="../search_identifications/" class="btn btn-xs btn-default pull-left">
        <i class="fa fa-chevron-left" aria-hidden="true"></i>
        Previous</a><a href="../search_identifications/" class="btn btn-xs btn-link">
        Search Identifications
      </a>
    </div>
    
  </div>

    <br>
</div>

<footer class="container-fluid wm-page-content">
      <p>
        <a href="https://github.com/CCMS-UCSD/MassIVEDocumentation/blob/master/docs/ptm_analysis.md">Edit on </a>
      </p>
  <p>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a> using <a href="https://github.com/gristlabs/mkdocs-windmill">Windmill</a> theme by Grist Labs.</p>
</footer>

</body>
</html>